Source/ChilliSource/Rendering/Material/Material.h

```diff
@@ -220,6 +238,61 @@ namespace ChilliSource
         /// @return Dest BlendMode of blending functions
         //----------------------------------------------------------
         BlendMode GetDestBlendMode() const;
+
+        /// @param The equation that describes how source and dest pixels are blended
+        ///
+        void SetBlendEqn(BlendEqn eqn) noexcept { m_blendEqn = eqn; }
```

Comment:
IIRC several things in the engine assuming that additive blend is used; is it a good idea to expose this to the user, potentially breaking that?